triangle_facts/triangle_facts.rb

```diff
@@ -1,45 +1,65 @@
 # Killer facts about triangles AWW YEAH
 class Triangle
-	attr_accessor :side1,:side2,:side3
+	attr_reader :side1, :side2, :side3
 
-	def initialize(side1,side2,side3)
-		@side1,@side2,@side3 = side1,side2,side3
+	def initialize(side1, side2, side3)
+		@side1, @side2, @side3 = side1, side2, side3
 	end
 
-	def equilateral()
-		return side1 == side2 && side2 == side3
+	def recite_facts
+		puts side_length_sentence
+		puts angle_sentences
 	end
 
-	def isosceles()
-		return [side1,side2,side3].uniq.length == 2
-	end
+	def calculate_angles(a, b, c)
+		angleA = calculate_single_angle(b, c, a)
+		angleB = calculate_single_angle(c, a, b)
+		angleC = calculate_single_angle(a, b, c)
 
-	def scalene()
-		unless ! (equilateral || isosceles)
-	    false
-	  else
-	    true
-	  end
+		[angleA, angleB, angleC]
+	end
+	
+	private
+	def equilateral
+		side1 == side2 && side2 == side3
 	end
 
-	def recite_facts
-		puts 'This triangle is equilateral!' if equilateral
-		puts 'This triangle is isosceles! Also, that word is hard to type.' if isosceles 
-		puts 'This triangle is scalene and mathematically boring.' if scalene 
+	def isosceles
+		[side1,side2,side3].uniq.length == 2
+	end
 
-		angles = self.calculate_angles(side1,side2,side3)
-		puts 'The angles of this triangle are ' + angles.join(',')
+	def scalene
+		(equilateral || isosceles) ? false : true
```

Comment:
Both `equilateral` and `isosceles` are booleans so you don't need the ternary. Can you remove it while keeping the logic the same?